From: Jiri Bohac <jbohac@suse.cz>
Subject: provide a pppol2tp module alias for l2tp_ppp
Patch-mainline: no

pppol2tp has been moved to l2tp_ppp ins SLE11 SP2. Provide a module alias to
the new module name.

--- a/net/l2tp/l2tp_ppp.c
+++ b/net/l2tp/l2tp_ppp.c
@@ -1838,3 +1838,4 @@ MODULE_AUTHOR("James Chapman <jchapman@katalix.com>");
 MODULE_DESCRIPTION("PPP over L2TP over UDP");
 MODULE_LICENSE("GPL");
 MODULE_VERSION(PPPOL2TP_DRV_VERSION);
+MODULE_ALIAS("pppol2tp");
